pkg/oss/nodeserver.go

```diff
@@ -161,20 +167,25 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		if err != nil {
 			return nil, status.Error(codes.Internal, err.Error())
 		}
-		ossfsMounter = mounter.NewOssCmdMounter(ns.ossfsPaths[opts.FuseType], req.VolumeId, ns.rawMounter)
+		ossfsMounter = mounter.NewForMounter(
+			mounter.NewOssCmdMounter(ns.ossfsPaths[opts.FuseType], req.VolumeId, ns.rawMounter),
+			ossInterceptors[opts.FuseType]...,
+		)
 	} else {
 		ossfsMounter = mounter.NewProxyMounter(socketPath, ns.rawMounter)
```

Comment:
Here it looks a bit odd. `Mounter` is a relatively high-level abstraction that includes both `CmdMounter` and `ProxyMounter`. If we define a function like `NewForMounter`, it should be used to create any type of mounter, instead `NewProxyMounter`.  Or we can still use the `NewOssCmdMounter`?

Reply:
```
		ossfsMounter = mounter.NewForMounter(
			mounter.NewProxyMounter(socketPath, ns.rawMounter),
		)
```